### DISCUSSION TITLE
Bet was PRICE_ABOVE_MARKET but stake is not returned?

### DISCUSSION THREAD

**Discussion by AKRD44** [cloudbet/docs#99] [Q&A] [answered]
Hi there, 

I was testing out sending a 1$ bet via api and its bet status was. 

{'referenceId': '701dd54c-e540-41d0-bf00-05e3968a6563', 'price': '2.55', 'eventId': '25956555', 'marketUrl': 'soccer.match_odds/away', 'side': 'BACK', 'currency': 'USDC', 'stake': '1.0', 'createTime': '2024-12-02T20:44:42Z', 'status': 'PRICE_ABOVE_MARKET', 'returnAmount': '0.0', 'eventName': '', 'sportsKey': 'soccer', 'competitionId': '778', 'categoryKey': 'italy', 'customerReference': '25a3cf90-a2c3-44fb-8656-b9653e300522', 'error': ''}

Is the order still "active" and waiting for the price to hit the one I asked or is it cancelled? I noticed that my 1$ was not returned.  If it is still "active", is there a way to cancel it? I don't see any method for that in the documentation.

Thanks

**Comment by kgravenreuth** [comment, marked as answer]
Hi @AKRD44 ,
Thanks for giving details. After checking your records, this bet was rejected and the staked amount was not deducted from your account as expected. Instead, there was another bet placed via the Cloudbet API at the same time which was indeed accepted. 

You can check your bet history via the Cloudbet Bet History API endpoint `/v4/bets/history` to find the `referenceId` of the bet which did successfully go through: https://www.cloudbet.com/api/?urls.primaryName=Trading#/Trading/BetHistory

Hope this will clarify your question. Let me know if you need any other pointers.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by AKRD44** [comment]
Ah I see, thanks Klaus!
